gf-webapp-admin/src/component/rooms/room/promo/Promo.tsx

```diff
@@ -34,7 +34,8 @@ export const Promo = (props: PromoProps) => {
 
   return (
     <div className={styles.promo}>
-      <SmallTitle
+      <Title
+        size="h4"
         text="Promo"
```

Comment:
Please don't use magic value

Reply:
"Promo" isn't a magic value

Reply:
I think that here promo is a classic magic value. Why don't u think so?